keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterHandler.kt

```diff
@@ -540,16 +540,19 @@ class TitusClusterHandler(
       containerAttributes = containerAttributes,
       iamProfile = iamProfile,
       migrationPolicy = migrationPolicy,
-      resources = resources.exportSpec(),
+      resources = resources.toSpec(),
       tags = tags
     )
 
     return checkNotNull(buildSpecFromDiff(defaults, thisSpec))
   }
 
-  private fun Resources.exportSpec(): ResourcesSpec? {
```

Comment:
can you explain what was wrong in the old implementation? did we not capture the right data?

Reply:
Well, it seemed like we were trying to only export the difference between the default for the resources field. but that didn't work. So just made it output the whole resources section there. TBH I don't understand how to debug the recursive diffing.